### ISSUE TITLE
Login via password now working

### ISSUE THREAD

**Issue by ykakkar42** [krreet/plugin.video.jiotv#104]
If i try to login to plugin via jio id and password its not working, i have been using this password method from years instead of otp, it was supported, but in latest tobalan version its not working.

**Comment by rrjanbiah** [comment]
@ykakkar42 

Looks like a duplicate of #102 ?

**Comment by technofreak24** [comment]
**Jio ID will not work**
If you have email address set on https://www.jio.com/selfcare/signup/
Use any of the following modes to login:
• Email Address and Password *(Auto-login after 21 days)*
• Jio Number and Password *(Auto-login after 21 days)*
or
• Jio Number and OTP *(Auto Token Refresh after 21 days)*

![IMG_20230810_160248_040528](https://github.com/krreet/plugin.video.jiotv/assets/121749085/d013f6bd-35fc-4b8e-a49f-75eba0483cd3)

**Comment by technofreak24** [comment]
**Jio ID will not work**
If you have email address set on https://www.jio.com/selfcare/signup/
Use any of the following modes to login:
• Email Address and Password *(Auto-login after every 21 days)*
• Jio Number and Password *(Auto-login after every 21 days)*
or
• Jio Number and OTP *(Auto-Token Refresh after every 21 days)*
![IMG_20230818_172017_052136](https://github.com/krreet/plugin.video.jiotv/assets/121749085/412bfd2e-df9c-4909-beb4-dcc8afbc73dd)
![IMG_20230816_221512](https://github.com/krreet/plugin.video.jiotv/assets/121749085/83d5d9bb-201d-40af-9bb6-79cbdbca18c3)
![IMG_20230810_160248_040528](https://github.com/krreet/plugin.video.jiotv/assets/121749085/d013f6bd-35fc-4b8e-a49f-75eba0483cd3)

**Comment by ykakkar42** [comment]
Yes it started with that topic but went into retaining phone number on the page so dont have to enter again, hence i started this separate issue.

**Comment by piscesbangalore** [comment]
> If i try to login to plugin via jio id and password its not working, i have been using this password method from years instead of otp, it was supported, but in latest tobalan version its not working.

I am not able to use id and password. Do you know of any solution
